src/components/inputs/styles.js

```diff
@@ -1,42 +1,54 @@
 import { css } from '@emotion/core';
 import vars from '../../../materials/custom-properties';
+import designTokens from '../../../materials/design-tokens';
+
+const getInputStyles = (props, theme) => {
+  const overwrittenVars = {
+    ...vars,
+    ...theme,
```

Comment:
basically how it works, is in your theme, you overwrite one of our choices. So for "colorGreen", you can specify "pink".